sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -14,7 +14,7 @@ output-folder: ../
 source-code-folder-path: ./src/generated
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/25616b92ccaec6c8bd3dd51f1312243bca88fe2d/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.3/TextAnalytics.json
 add-credentials: false
-package-version: 5.1.0-beta.3
+package-version: 5.1.0-beta.4
```

Comment:
not related to this PR but I wonder if JS has a script or automated task to update the version of packages once you release?
I know this is true for .NET. WE get an automatic PR that bumps the version everywhere and updates the changelog

Reply:
we do too but I do not remember why that was not merged the last time.